app/src/main/java/com/nilhcem/blenamebadge/ui/main/MainActivity.kt

```diff
@@ -66,12 +65,14 @@ class MainActivity : AppCompatActivity(), PreviewChangeListener {
         fab_main.setOnClickListener {
             if (BluetoothAdapter.getDefaultAdapter().isEnabled) {
                 // Easter egg
-                fab_main.visibility = View.GONE
+                Toast.makeText(this, "Sending Data.", Toast.LENGTH_LONG).show()
```

Comment:
I would extract the string in here. Rest looks good, the animation is really cool

Reply:
Use string resources instead of hardcoding the string.

Reply:
done. have a look